src/main/java/com/stripe/model/SubscriptionCollection.java

```diff
@@ -0,0 +1,4 @@
+package com.stripe.model;
+
+public class SubscriptionCollection extends StripeCollection<Customer> {
```

Comment:
hm I believe the Java tests just verify that the correct call to the API is made but does not actually execute the request and render the response